l10n_es_facturae/data/Facturaev3_2.xsd

```diff
@@ -1670,8 +1670,8 @@
 	</xs:complexType>
 	<xs:complexType name="ExtensionsType">
 		<xs:sequence>
-			<xs:any namespace="##other" processContents="strict" minOccurs="0" maxOccurs="unbounded"/>
-		</xs:sequence>
+            <xs:any namespace="##other" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
I don't get why this change.

Reply:
ping @etobella

Reply:
Esto estaba en otro commit para la 14. La idea de este cambio es permitir añadir extensiones. Seria requerido, por ejemplo, en el caso de enviar facturas a FaceB2B

Reply:
Vale, pero esto no va en el mismo commit y debe llevar su propia explicación y que el título del PR y comentario principal se adapte también.